scanapi/template_render.py

```diff
@@ -20,3 +21,12 @@ def _loader(is_external):
         return FileSystemLoader(searchpath="./")
 
     return PackageLoader("scanapi", "templates")
+
+
+def render_body(request):
```

Comment:
Would it be possible to have tests for this? The test file for this part is https://github.com/scanapi/scanapi/blob/master/tests/unit/test_template_render.py

Here you can find a brief guideline to help you with the tests: https://github.com/scanapi/scanapi/wiki/Run-ScanAPI-Locally#tests